charts/rancher-external-dns/v0.1.1/templates/deployment.yaml

```diff
@@ -80,6 +80,9 @@ spec:
           {{- if .Values.google.project }}
             - --google-project={{ .Values.google.project }}
           {{- end }}
+          {{- if eq .Values.provider "aws" }}
+            - --aws-batch-change-size=100
+          {{- end }}
```

Comment:
Instead of setting this value to 100, we should make it configurable by exposing in Values.yaml and set to 100 in Values

```
          {{- if eq .Values.provider "aws" }}
            - --aws-batch-change-size={{ .Values.aws.batchChangeSize }}
          {{- end }}
```

Reply:
Agree, thanks for the suggestion